src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalInternalUserFunctionalTest.java

```diff
@@ -1231,4 +1233,80 @@ private static List<Map<String, Object>> sortByValue(final List<Map<String, Obje
                 .sorted(Comparator.comparing(map -> (String) map.get(key)))
                 .collect(Collectors.toList());
     }
+
+    @Test
+    @ToggleEnable(mapKey = "OrganisationExternalController.deletePaymentAccountsOfOrganisation", withFeature = true)
+    @ExtendWith(FeatureToggleConditionExtension.class)
+    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+    void deletePbaOfExistingOrganisationShouldBeForbiddenWhenLDOff() {
+        log.info("deletePbaOfExistingOrganisationShouldBeForbiddenWhenLDOff :: STARTED");
+
+        setUpTestData();
+        String bearerToken = inviteUser(puiFinanceManager, intActiveOrgId, generateRandomEmail(),
```

Comment:
dont need to invite another user to delete payment account ? for internal endpoint prd-admin only allowed to delete payment account.

Reply:
done